packages/components/tsconfig.json

```diff
@@ -21,6 +21,6 @@
             "#/*": ["./src/*"]
         }
     },
-    "include": [".storybook", "src"],
+    "include": [".storybook", "src", "scripts"],
```

Comment:
Should be added in the `tsconfig.node.json`

Reply:
switched